{"name": "test_true", "status": "failed", "statusDetails": {"message": "AssertionError: assert False == True\n  +False\n  -True", "trace": "self = <test_leap_year.TestAssert object at 0x000001A393E93708>\n\n    def test_true(self):\n>       assert is_leap_year(2017) == True\nE       assert False == True\nE         +False\nE         -True\n\ntest_leap_year.py:33: AssertionError"}, "start": 1588052546300, "stop": 1588052546300, "uuid": "05788c9f-68c9-4fc4-87b1-df1f9d6c324a", "historyId": "9db7ba8602f272ed967dcdafbb3f3521", "testCaseId": "34603d2208cc04ae1d25f5b0c05a0fa3", "fullName": "test_leap_year.TestAssert#test_true", "labels": [{"name": "suite", "value": "test_leap_year"}, {"name": "subSuite", "value": "TestAssert"}, {"name": "host", "value": "PC-BFD8TY2"}, {"name": "thread", "value": "3208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_leap_year"}]}